internal/services/policy/client/client.go

```diff
@@ -30,7 +30,7 @@ func NewClient(o *common.ClientOptions) *Client {
 	setDefinitionsClient := policy.NewSetDefinitionsClientWithBaseURI(o.ResourceManagerEndpoint, o.SubscriptionId)
 	o.ConfigureClient(&setDefinitionsClient.Client, o.ResourceManagerAuthorizer)
 
-	policyInsightsClient := policyinsights.NewPolicyInsightsClientWithBaseURI(o.ResourceManagerEndpoint)
+	policyInsightsClient := remediations.NewRemediationsClientWithBaseURI(o.ResourceManagerEndpoint)
```

Comment:
Minor but for consistency
```suggestion
	remediationsClient := remediations.NewRemediationsClientWithBaseURI(o.ResourceManagerEndpoint)
```